subsys/bootloader/Kconfig

```diff
@@ -105,6 +105,18 @@ config SB_DEBUG_SIGNATURE_PUBLIC_KEY_LAST
 	  first. This is meant to be used for testing looping through the
 	  public keys.
 
+config SB_NUM_MONOTONIC_COUNTERS
+	int "Number of monotonic counters"
+	default 0
```

Comment:
As mentioned elsewhere, could we set some limit here to reflect the limitations in hardware, or is this check so complex that we maybe need to do it in cmake? For instance on 91 the otp region size is limiting, but on 52 the size of the provision partition is the limiting factor.